wire-ios-protos/fastlane/Fastfile

```diff
@@ -0,0 +1 @@
+import_from_git(url: 'https://github.com/wireapp/wire-ios-shared-resources.git', branch: "master", path: "Fastlane/Frameworks")
```

Comment:
PR https://github.com/wireapp/wire-ios/pull/2217 accidentally deleted this file.

Reply:
well we don't need it since this is dead code